docs/README.md

```diff
@@ -27,6 +27,7 @@ These documents provide you with more architectural details:
 - [Proposal: API Packages](./architecture/api-packages.md)
 - [Proposal: Input parameters for API, Event Definitions or API Packages in the Runtime](./architecture/input-parameters-on-runtime.md)
 - [Proposal: Custom configuration for Kyma installation](./architecture/provisioning-custom-configuration.md)
+- [Proposal: Pairing Application Instance with Compass](./architecture/pairing-application-instance-with-compass.md)
```

Comment:
General comment: 
from my perspective, paring application and pairing application managed by the integration system are two completely different things:
- in the first case, pairing means establishing a trusted connection between Application Instance and Compass Director
- in the second case, pairing means establishing a trusted connection between Application Instance and Integration System.

For me, it was extremely confusing and we should show via our API this difference.